core/engine/src/vm/opcode/call/mod.rs

```diff
@@ -340,69 +386,49 @@ async fn load_dyn_import(
     // 3. Let loadPromise be module.LoadRequestedModules().
     let load = module.load(&mut context.borrow_mut());
 
-    // 4. Let rejectedClosure be a new Abstract Closure with parameters (reason) that captures promiseCapability and performs the following steps when called:
```

Comment:
question: why are the specification steps being removed?

Are these no longer applicable? If they're not applicable, what are the new steps?